apps/omg/config/config.exs

```diff
@@ -6,7 +6,8 @@ config :omg,
   deposit_finality_margin: 10,
   ethereum_events_check_interval_ms: 8_000,
   coordinator_eth_height_check_interval_ms: 6_000,
-  metrics_collection_interval: 60_000
+  metrics_collection_interval: 60_000,
+  fee_claimer_address: "NO FEE CLAIMER ADDR!"
```

Comment:
if this is a configuration value that needs to be set before the application is deployed I suggest using `ReleaseTasks` (don't forget to let devops know of this requirement and updating the docs)

Reply:
Good point @T-Dnzt @InoMurko - especially Release tasks!
We also have demo/play-with mode where one can set `:fees_not_required` instead of `fee json`. Because surpluses are collected any way regardless this setting there should be some non-zero `fee claimer` address (can't be zero because of validation rules when tx gets recovered from binary)... For now I can make this config value obligatory.